glossary/en/terms/open-government/index.md

```diff
@@ -4,4 +4,4 @@ lang: en
 title: Open government
 ---
 
-Open government data, in line with the open movement generally, seeks to make the workings of governments transparent, accountable, and responsive to citizens. It includes the ideals of democracy, due process, citizen participation and open government data. A thorough-going approach to open government would also seek to enable citizen participation in, for example, the drafting and revising of legislation and budget-setting. See [OGP](/glossary/en/terms/ogp/).
+Open government, in line with the open movement generally, seeks to make the workings of governments transparent, accountable, and responsive to citizens. It includes the ideals of democracy, due process, citizen participation and open government data. A thorough-going approach to open government would also seek to enable citizen participation in, for example, the drafting and revising of legislation and budget-setting. See [OGP](/glossary/en/terms/ogp/).
```

Comment:
I think this whole term is confusing. Open government is a term that was way know before the open movement and is based in the 18th Century. I will accept this change, but I think we should be clearer here (and thank you for bringing it to our attention!)